snapshot-labs/sx-monorepo
# feat: space user profile page

### Summary



This PR add a new user profile page, under each space, summarizing the user activities for the given space

In order to facilitate reviewing, this feature will be split into multiple PR, and this one will only handle the skeleton and routes to all pages, as well as the everything related to the `STATEMENT` tab. All other pages (delegators, proposals and votes) will be empty, and populating them will come in a future PR.

### How to test

1. Go to a space leaderboard
2. Click on an user
3. It should redirect you to the space user profile page
4. For onchain spaces, it should show the correct number of proposals and votes
5. It should display the user's voting power on this space
6. It should show the user's statement if available
7. When it's your own profile, you should be able to edit it

### TODO 

- [x] Display voting power
- [x] Edit statement
- [x] Add new fields (`status`, `discourse`) to SDK - https://github.com/snapshot-labs/snapshot.js/pull/1023
- [x] Add new fields (`status`, `discourse`) to Sequencer - https://github.com/snapshot-labs/snapshot-sequencer/pull/386
- [x] Add new fields (`status`, `discourse`) to Hub - https://github.com/snapshot-labs/snapshot-hub/pull/870
